Classes/Runtime/FusionObjects/RuntimeFormImplementation.php

```diff
@@ -100,7 +100,9 @@ public function evaluate(): string
         $process = $this->getProcess();
 
         $formRequest = $this->formRequestFactory->createFormRequest($this->getCurrentActionRequest(), $namespace);
-        $this->runtime->pushContext('request', $formRequest);
+        $context = $this->runtime->getCurrentContext();
+        $context['request'] = $formRequest;
+        $this->runtime->pushContextArray($context);
```

Comment:
please add a big comment that this is not recommended and we cannot use `pushContext` to achieve the same ^^

Reply:
Not really needed, fusion form uses `pushContextArray` in other places already 🤷‍♂️

Reply:
I dont want to loathe the method `pushContextArray` itself as this "internal" use case is fine. I just want to document that we are doing something "unwise" here by replacing "request" and that no one should follow this lead ;)

```suggestion
        $context = $this->runtime->getCurrentContext();
        /**
          * The internal method "pushContextArray" allows some creative use,
          * as that we can override the "request" context.
          * This is not permitted via public / official api and probably an unwise idea to do.
          * {@see \Neos\Fusion\Core\FusionGlobals}
          */
        $context['request'] = $formRequest;
        $this->runtime->pushContextArray($context);
```